src/quo2/components/wallet/network_bridge/view.cljs

```diff
@@ -0,0 +1,45 @@
+(ns quo2.components.wallet.network-bridge.view
+  (:require
+    [clojure.string :as string]
+    [quo2.components.icon :as icon]
+    [quo2.components.markdown.text :as text]
+    [quo2.foundations.colors :as colors]
+    [quo2.foundations.resources :as resources]
+    [quo2.theme :as quo.theme]
+    [react-native.core :as rn]
+    [quo2.components.wallet.network-bridge.style :as style]))
+
+(defn network-bridge-internal
+  [{:keys [theme network state amount]}]
+  (let [network-text (if (= network :ethereum) "Mainnet" (string/capitalize (name network)))]
+    [rn/view
+     {:style               (style/container network state)
+      :accessibility-label :container}
+     (if (= state :loading)
```

Comment:
@OmarBasem, @J-Son89, I was under the impression we wanted to use the same names in Figma to avoid the constant impedance mismatch between code and Figma.

This means the component should accept the property `status`, not `state`.

Reply:
Yep let's match whatever's in figma 👍

Reply:
Fixed it, thanks @ilmotta 👍